test-common/src/main/java/com/gentics/mesh/test/docker/MeshContainer.java

```diff
@@ -690,7 +691,7 @@ public String getContainerIpAddress() {
 		if (containerHost != null) {
 			return containerHost;
 		} else {
-			return super.getContainerIpAddress();
+			return DockerClientFactory.instance().dockerHostIpAddress();
```

Comment:
This makes a stack overflow.

Reply:
Maybe it would be better to move the whole implementation to getHost() and do not overwrite the deprecated getContainerIpAddress() at all?